app/models/manageiq/providers/google/cloud_manager/vm/operations/guest.rb

```diff
@@ -1,6 +1,12 @@
 module ManageIQ::Providers::Google::CloudManager::Vm::Operations::Guest
-  def validate_reboot_guest
-    validate_vm_control_powered_on
+  extend ActiveSupport::Concern
+  include SupportsFeatureMixin
+
+  included do
+    supports :reboot_guest do
+      unsupported_reason_add(:reboot_guest, unsupported_reason(:control)) unless supports_control?
+      unsupported_reason_add(:reboot_guest, _("The VM is not powered on")) unless current_state == "on"
```

Comment:
Prefer `N_` for i18n strings with `unsupported_reason_add` as they need to be translated on the fly if the target language changes.